### ISSUE TITLE
Add support for editing metadata for existing chunks

### ISSUE THREAD

**Issue by deshraj** [embedchain/embedchain#1078]
### 🚀 The feature

Currently, if you ingest some data and want to attach some metadata after indexing the chunks, it is not easy to do that and a developer would have to update the metadata manually.

Instead, we want to be able to edit the metadata (add new key, edit existing key, delete key) from vector store with a simple interface.

### Motivation, pitch

Motivation is to make sure that the user doesn't have to index the data twice if they only want to update the metadata for the indexed data.

**Comment by SeeYangZhi** [comment]
@deshraj Is this the main blocker for why https://api.mem0.ai/v1/memories/{memory_id}/ does not support updating metadata? Looking at https://docs.mem0.ai/api-reference/memory/update-memory, currently it seems to only support the update of the text content of the memory.

**Comment by deshraj** [comment]
Yes this has come up recently when talking to other developers. We will ship this feature soon. cc @prateekchhikara

**Comment by SeeYangZhi** [comment]
Looks like the feature has been shipped in the latest version https://github.com/mem0ai/mem0/releases/tag/v0.1.109
With PR https://github.com/mem0ai/mem0/pull/2976 by @prateekchhikara 

Docs: https://docs.mem0.ai/platform/quickstart#4-6-update-memory
Thanks!
